src/components/ButtonPanel.jsx

```diff
@@ -12,8 +12,14 @@ const buttonsLabel = [
 ];
 
 const ButtonGroup = ({ group, id }) => {
-  const buttonGroups = group.map(label => (
-    <Button key={label} name={label} isWide={label === '0'} />
+  const buttonGroups = group.map((label, index) => (
+
+    <Button
+      key={label}
+      color={index < group.length - 1 ? 'lightgrey' : 'orange'}
```

Comment:
this is sending the orange. the default props is not the one add the background. this is fine anyway.